code/standards/checkstyle.xml

```diff
@@ -41,22 +47,24 @@
     <module name="MethodParamPad"/>
     <module name="NoWhitespaceAfter"/>
     <module name="NoWhitespaceBefore"/>
-    <module name="OperatorWrap"/>
     <module name="ParenPad"/>
     <module name="TypecastParenPad"/>
     <module name="WhitespaceAfter"/>
     <module name="WhitespaceAround">
       <property name="tokens" value="ASSIGN"/>
     </module>
     <module name="ModifierOrder"/>
-    <module name="AvoidNestedBlocks"/>
+    <module name="AvoidNestedBlocks">
+      <property name="allowInSwitchCase" value="true" />
```

Comment:
My personal opinion is switch is generally not what we should be using - those should strongly be considered to be situations where we should ask if something should actually have a method.  A really complicated switch is a code smell, IMHO.  So I personally would not want to encourage large switch statements by having blocks.